mobile-app/app/components/EstimatedFeeInfo.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { tailwind } from '@tailwind'
+import { ThemedIcon, ThemedText, ThemedView } from './themed'
+import { BottomSheetModal } from './BottomSheetModal'
+import { translate } from '@translations'
+import { View } from 'react-native'
+import NumberFormat from 'react-number-format'
+
+interface EstimatedFeeElement {
+  value: string | number
+  suffix?: string
+  testID: string
+}
+
+export function EstimatedFeeInfo ({
+  lhs,
+  rhs
+}: { lhs: string, rhs: EstimatedFeeElement }): JSX.Element {
+  return (
+    <ThemedView
+      dark={tailwind('bg-gray-800 border-b border-gray-700')}
+      light={tailwind('bg-white border-b border-gray-200')}
+      style={tailwind('p-4 flex-row items-start w-full')}
+    >
+      <View style={tailwind('w-5/12')}>
+        <View style={tailwind('flex-row items-center justify-start')}>
+          <ThemedText style={tailwind('text-sm mr-1')} testID={`${rhs.testID}_label`}>
+            {lhs}
+          </ThemedText>
+
+          <BottomSheetModal
+            name='EstimatedFeeInfo'
+            snapPoints={['50%', '50%']}
+            triggerComponent={
+              <ThemedIcon
+                size={16}
+                name='info'
+                iconType='MaterialIcons'
+                light={tailwind('text-warning-500')}
+                dark={tailwind('text-darkwarning-500')}
+              />
+          }
+          >
+            <View style={tailwind('p-4')}>
+              <View
+                testID='estimated_fee_heading'
+                style={tailwind('flex-row mb-3 items-center')}
+              >
+                <ThemedIcon
+                  size={24}
+                  name='info'
+                  iconType='MaterialIcons'
+                  light={tailwind('text-warning-500')}
```

Comment:
Figma doesnt have a color on the info icon

Reply:
For consistency of info icon I have used yellow colour